tests/validators/test_int.py

```diff
@@ -26,12 +26,18 @@
         ('42', 42),
         (42.0, 42),
         ('42.0', 42),
+        ('42.00', 42),
         ('042', 42),
         ('4_2', 42),
         ('4_2.0', 42),
         ('04_2.0', 42),
+        ('  04_2.0 ', 42),
+        # because zeros are striped before underscores this is not allowed
+        ('  0_42.0 ', Err('Input should be a valid integer, unable to parse string as an integer')),
```

Comment:
I noticed this slightly unfortunate but expected format that will fail validation, and added a test to document it. I don't think it needs to be fixed.